library/api.inc

```diff
@@ -1,6 +1,6 @@
 <?php
 //our api for 3rd party developers
-include_once("../../globals.php");
+include_once("/../interface/globals.php");
```

Comment:
You shouldn't use _/.._ because this will create the path from **root**

![image](https://user-images.githubusercontent.com/6632800/34645451-9a682358-f345-11e7-9ae5-cf9b7866af7e.png)

Reply:
Thanks @MigDinny